package.json

```diff
@@ -7,18 +7,18 @@
     "@commitlint/config-conventional": "^13.1.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^11.0.0",
+    "@commitlint/cli": "^13.1.0",
     "eslint": "^7.11.0",
     "eslint-config-6river": "^5.0.0",
-    "husky": "*"
+    "husky": "^7.0.0"
   },
   "peerDependencies": {
-    "@commitlint/cli": "^11.0.0",
-    "husky": "*"
+    "@commitlint/cli": "*"
```

Comment:
This is listed in `devDependencies` _and_ `peerDependencies`.  Can it just be kept as a dev dep and removed from peer deps?

Reply:
It needs to be in `devDependencies` so that this repository can use it, and `peerDependencies` so that consumers know they need to install it.

Reply:
Ah I see what you're saying, we're linting commits and we also want consumers to know that they need that package for this to work.  Are we truly ok with _any_ commitlint version?  Should it just be `>11.0.0` ?

Reply:
Yeah, I've been blindly updating the CLI for quite a while at this point and have never encountered an incompatibility between the config and the CLI.

Our config doesn't add much to `@commitlint/config-conventional`, so maybe we've just been lucky and the underlying package has been making changes to stay compatible - I'm not sure. That package doesn't have a peer dependency on the CLI, so it doesn't provide any guidance for what we should do here.

You're right, it would be safer to limit the minimum version! I'll do that.